docs/src/reference-manual/cbmc-starter-kit-setup-ci.md

```diff
@@ -0,0 +1,58 @@
+# Reference manual
+
+## Name
+
+`cbmc-starter-kit-setup-ci` - Add GitHub Actions workflow to run proofs as part of CI
+
+## Synopsis
+
+```
+cbmc-starter-kit-setup-ci [-h] --github-actions-runner
+                                 <ubuntu-20.04>|<name-of-your-Ubuntu-20.04-large-runner>
+                                 [--cbmc <latest>|<X.Y.Z>]
```

Comment:
I don't see a good reason for these lines to be indented this far. (And they don't even match up with `--github-actions-runner`. Can each flag be indented by just two spaces.

Reply:
I will indent by 2 spaces.